crates/ruff_server/docs/setup/NEOVIM.md

```diff
@@ -54,3 +54,35 @@ require('lspconfig').pyright.setup {
   },
 }
 ```
+
+By default, Ruff will not show any logs. To enable logging in Neovim, you'll need to set the `RUFF_TRACE` environment variable
+to either `messages` or `verbose`:
+
+```lua
+require('lspconfig').ruff.setup {
+  cmd_env = { RUFF_TRACE = "messages" }
+}
+```
+
+You can set the log level in `settings`:
+
+```lua
+require('lspconfig').ruff.setup {
+  cmd_env = { RUFF_TRACE = "messages" },
+  settings = {
+    logLevel = "debug",
+  }
```

Comment:
I don't think you can directly provide the `settings` key at the top level because the [new server doesn't support `workspace/didChangeConfiguration` (yet)](https://github.com/astral-sh/ruff/blob/84e87771cba841c08ac7d5d1017050b8bd22837e/crates/ruff_server/src/server/api/notifications/did_change_configuration.rs#L20-L21). So, the correct code should be to specify it under `init_options` which sends this in the initialize request:

```suggestion
  init_options = {
    settings = {
      logLevel = "debug",
    }
  }
```